en/docs/adv_examples/resnet.md

```diff
@@ -1,373 +1,353 @@
 ```
-train: epoch 0, iter 200, loss: 7.024337, top_1: 0.000957, top_k: 0.005313, samples/s: 964.656
-train: epoch 0, iter 400, loss: 6.849526, top_1: 0.003594, top_k: 0.012969, samples/s: 991.474
-... train: epoch 0, iter 5000, loss: 5.557458, top_1: 0.064590, top_k: 0.174648, samples/s: 935.390
-Saving model to ./output/snapshots/model_save-20200629223546/snapshot_epoch_0. validation: epoch 0, iter 100, top_1: 0.074620, top_k: 0.194120, samples/s: 2014.683
-```
-
-可以看到：
-
-- 随着训练的进行，loss不断下降，而训练的top_1/top_k准确率不断提高（其中top_k默认为top_5准确率，可自定义）。
-- 每个epoch结束时，会做另外两个工作：1）执行一次验证，并打印出验证集上的top_1/top_k准确率；2）保存模型。
-- samples/s 用来指示训练/验证的执行速度，即每秒钟能处理的图片数量。
 
+> Before executing `sh evaluate.sh`, make sure you have prepared the validation dataset of ImageNet 2012. Please refer to [OneFlow-Benchmark](https://github.com/Oneflow-Inc/OneFlow-Benchmark/blob/master/Classification/cnns) repository to learn how to make validation dataset. 
 
+From the evaluation results of the three rounds, out model has achieved 77.32+% Top1 accuracy. 
 
-**复现实验的说明：**
+Finally, congratulations! You complete the training / validating, inference and evaluation of ResNet model on ImageNet dataset. Applause for yourself!
 
-> Q1. 多久能够完成训练？
-> 
-> 在GPU环境下，使用单机8卡（NVIDIA TITAN V），完成90个epoch的完整训练过程，大概需要15小时。
-> 
-> 中，提供ResNet50 v1.5的OneFlow实现。我们在ImageNet-2012数据集上训练90轮后，验证集上的准确率能够达到：77.318%(top1)，93.622%(top5)。
-> 
-> 训练集：80.57%（top1）
-> 
-> 验证集：77.318%（top1），93.622%（top5）
 
 
+## Details
 
-## 更详细的说明 Details
+### Distributed training
 
-### 分布式训练
+**Simple and easy-to-use distributed training is one of OneFlow's main features**
 
-**简单而易用的分布式，是OneFlow的主打特色之一。**
+OneFlow is designed to support efficient distributed training natively. Especially for distributed data parallelism, user do not have to worry about how to divide and synchronize the data when the algorithm expands from single machine to multiple machines. That is to say, in OneFlow, User only need to write algorithm from the view of single machine, and the code automatically has the ability of distributed training. 
 
-OneFlow框架从底层设计上，就原生支持高效的分布式训练。尤其对于分布式的数据并行，用户完全不用操心算法从单机单卡扩展到多机多卡时，数据如何划分以及同步的问题。OneFlow框架从底层设计上，就原生支持高效的分布式训练。尤其对于分布式的数据并行，用户完全不用操心算法从单机单卡扩展到多机多卡时，数据如何划分以及同步的问题。也就是说，使用OneFlow，用户以单机单卡的视角写好算法，**自动具备多机多卡分布式数据并行的能力。**</strong>
 
+#### How to configure and run distributed training?
 
-#### 如何配置并运行分布式训练？
+We still use the code shown in the "Quick Start", in `train.sh`, the distributed configuration is easily accomplished by specifying the number of nodes (machines) with `--num_nodes`, the IP address of the nodes with `--node_ips`, and the number of devices to be used on each node with `--gpu_num_per_node`. 
 
-还是以上面"快速开始"部分演示的代码为例，在`train.sh`中，只要用`--num_nodes` 指定节点（机器）个数，同时用`--node_ips`指定节点的ip地址，然后用`--gpu_num_per_node`指定每个节点上使用的卡数，就轻松地完成了分布式的配置。
-
-例如，想要在2机8卡上进行分布式训练，像下面这样配置：
+For example, we want to do distributed training on  2 machines with 8 devices, configure it like this: 
```

Comment:
we want to do distributed training on  2 machines with 8 devices, configure it like this:
->
if we want to run distributed training on 2 machines with 8 devices, we will configure it like this: